pkg/pb/orders.proto

```diff
@@ -46,6 +46,8 @@ message OrderLimit {
     // satellites aren't necessarily discoverable in kademlia. this allows
     // a storage node to find a satellite and handshake with it to get its key.
     node.NodeAddress satellite_address = 11;
+
+    google.protobuf.Timestamp order_creation = 12 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
```

Comment:
Can you make a separate PR that changes this for all fields? Also, maybe sort it near other timestamps.